lib/Target/TTNN/TTNNToFlatbuffer.cpp

```diff
@@ -74,11 +74,18 @@ createOp(FlatbufferObjectCache &cache, CloseDeviceOp op) {
 
 ::flatbuffers::Offset<::tt::target::ttnn::ToMemoryConfigOp>
 createOp(FlatbufferObjectCache &cache, ToMemoryConfigOp op) {
-  auto input = getOperandThroughDPSOps(op.getInput());
-  auto output = getOperandThroughDPSOps(op.getOutput());
-  return ::tt::target::ttnn::CreateToMemoryConfigOp(
-      *cache.fbb, cache.at<::tt::target::TensorRef>(input),
-      cache.at<::tt::target::TensorRef>(output));
+  constexpr uint64_t kHostAllocatedAddress = 0;
+  constexpr uint64_t kHostAllocatedSize = 0;
+  auto input = cache.getOrCreate(
+      op.getInput(), tensorValueToFlatbuffer, kHostAllocatedAddress,
+      kHostAllocatedSize); // TODO: Fix this so it reads an existing value - one
+                           // should always exist in cache. This is a temporary
+                           // workaround until
+                           // https://github.com/tenstorrent/tt-mlir/issues/435
```

Comment:
See my comment on the issue, I think it has to work this way, so maybe we can remove this TODO?